java/test/jmri/jmrit/logix/LearnWarrantTest.java

```diff
@@ -99,7 +99,8 @@ public void testLearnWarrant() throws Exception {
         Sensor lastSensor = recordtimes(route, frame._speedUtil.getThrottle());
 
         // After stopping train, wait a bit before pressing stop
-        new org.netbeans.jemmy.QueueTool().waitEmpty(100);
+        JUnitUtil.waitFor(100);     // waitEmpty(100) causes a lot of failures on Travis GUI
```

Comment:
I'd be inclined to make this same change at other places in this test as well:

https://github.com/JMRI/JMRI/pull/9584/files#diff-af5ede92e8ff0141ebb58105f146b0f67c1c5b6caac94287363a0045ba6fc205R134

https://github.com/JMRI/JMRI/pull/9584/files#diff-af5ede92e8ff0141ebb58105f146b0f67c1c5b6caac94287363a0045ba6fc205R177

https://github.com/JMRI/JMRI/pull/9584/files#diff-af5ede92e8ff0141ebb58105f146b0f67c1c5b6caac94287363a0045ba6fc205R188

and

https://github.com/JMRI/JMRI/pull/9584/files#diff-af5ede92e8ff0141ebb58105f146b0f67c1c5b6caac94287363a0045ba6fc205R202

Reply:
> I'd be inclined to make this same change at other places in this test as well:

The links didn't worked, but I think I have fixed all of them.